app/src/main/java/org/liberty/android/fantastischmemo/entity/MultipleChoiceCard.java

```diff
@@ -6,8 +6,12 @@
 
 import org.liberty.android.fantastischmemo.dao.MultipleChoiceDaoImpl;
 
+import java.io.Serializable;
+
 @DatabaseTable(tableName = "multipleChoiceCards", daoClass = MultipleChoiceDaoImpl.class)
-public class MultipleChoiceCard  {
+public class MultipleChoiceCard implements Serializable{
+
+    private static final long serialVersionUID = 2112975910834873118L;
```

Comment:
What's this serialVersionUID used for? Why do the multiple choice cards need to be serializable?

Reply:
I pass a multiple choice card object in my intent in the recycle view adapter. To pass an object you need to use Serializable. I need to create an id because the static analysis tool said it was required.